src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -175,6 +175,7 @@ public void Version_MatchesFixedVersion()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
         public void WorkingSet_Valid()
```

Comment:
With my change on Uap, Environment.WorkingSet tries to get process information from process counter and then throws `System.InvalidOperationException : Process performance counter is disabled, so the requested operation cannot be performed.`